package.json

```diff
@@ -5,26 +5,23 @@
   "bin": "./bin/confidant",
   "license": "MPL-2.0",
   "main": "./build/main",
-
```

Comment:
I usually install package through command line:
```
$ npm install --save nconf
```
so npm is so smart to know where is the right position to put ```"nconf": "^0.7.1"```  in package.json automatically.
It also arrange package.json and remove redundant newlines.